src/components/blocks/FormBlock.tsx

```diff
@@ -28,19 +23,19 @@ const FormBlock: React.FC = () => {
     const closeShowModal = () => {
         dispatch(setOpenForm(false))
     }
-    const create = () => {
-        dispatch(createMovie())
+    const create = (movie) => {
+        dispatch(createMovie(movie))
     }
-    const update = () => {
-        dispatch(updateMovie())
+    const update = (movie) => {
+        dispatch(updateMovie(movie))
```

Comment:
Up to you, but these functions are not really necessary. You can use `dispatch(createMovie(movie))` and `dispatch(updateMovie(movie))`

Reply:
removed